CMakePresets.json

```diff
@@ -88,140 +121,73 @@
-      }
-    },
-    {
-      "name": "x64-static-debug",
-      "displayName": "Windows x64 Debug",
-      "description": "Windows x64 Debug",
-      "inherits": "x64-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug"
-      }
-    },
-    {
-      "name": "x64-static-release",
-      "displayName": "Windows x64 Release, static",
-      "description": "Windows x64 Release, static",
-      "inherits": "x64-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo"
+        "BUILD_TESTING": true,
+        "ENABLE_PROXY_TESTS": false,
+        "BUILD_SAMPLES": true
       }
     },
     {
-      "name": "x86-static-debug",
-      "displayName": "Windows x86 Debug, static",
-      "description": "Windows x86 Debug, static",
-      "inherits": "x86-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug"
-      }
-    },
-    {
-      "name": "x86-static-release",
-      "displayName": "Windows x86 Release, static",
-      "description": "Windows x86 Release, static",
-      "inherits": "x86-static",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "RelWithDebInfo"
-      }
+      "name": "x86-msvc-static-debug",
+      "displayName": "Windows x86, msvc static, Debug ",
+      "description": "Windows x86, msvc static, Debug",
+      "inherits": [ "x86-msvc-static", "debug-build" ],
+      "hidden": true
     },
     {
       "name": "x86-static-debug-tests",
       "displayName": "x86 Debug With Tests, static",
       "description": "Windows x86 Debug build with Tests configured",
-      "inherits": "x86-static-debug",
-      "cacheVariables": {
-        "BUILD_TESTING": true,
-        "ENABLE_PROXY_TESTS": false
-      }
+      "inherits": [ "x86-static", "debug-build", "enable-tests" ]
     },
     {
       "name": "x86-static-release-tests",
       "displayName": "x86 Release With Tests, static, libcurl",
       "description": "Windows x86 Release build with Tests configured",
-      "inherits": "x86-static-release",
+      "inherits": [ "x86-static", "release-build", "enable-tests" ],
       "cacheVariables": {
-        "BUILD_TESTING": true,
-        "ENABLE_PROXY_TESTS": false,
-        "BUILD_SAMPLES": true,
-        "BUILD_PERFORMANCE_TESTS": true,
-        "RUN_LONG_UNIT_TESTS": true,
-        "BUILD_TRANSPORT_CURL": true
+        "RUN_LONG_UNIT_TESTS": true
       }
     },
     {
       "name": "x64-debug-tests",
-      "displayName": "x64 Debug With Tests, winhttp+curl",
+      "displayName": "x64 Debug With Tests",
       "description": "Windows x64 Debug build with Tests configured",
-      "inherits": "x64-debug",
-      "cacheVariables": {
-        "BUILD_TESTING": true,
-        "ENABLE_PROXY_TESTS": false,
-        "BUILD_SAMPLES": true,
-        "BUILD_PERFORMANCE_TESTS": true,
-        "BUILD_TRANSPORT_WINHTTP": true,
-        "BUILD_TRANSPORT_CURL": true
```

Comment:
This preset is no longer setting winhttp/curl to true. It's a bit hard to follow in a diff'd review, so can you point me to the name of the preset which now has this, since we removed `winhttp+curl` from this one's name?

Reply:
x64-static-debug-perftests has both curl and winhttp. It's basically the kitchen sink (similar to x64-debug-tests).

Reply:
x64-debug-tests is exactly what it says it is. x64, debug, tests (the old name was wrong since it included perf tests, and samples).